bundles/core/org.eclipse.smarthome.core.thing/src/main/java/org/eclipse/smarthome/core/thing/binding/ThingHandler.java

```diff
@@ -174,4 +175,18 @@
      */
     void handleRemoval();
 
+    /**
```

Comment:
you don't need to really include a javadoc link. Just creating a connection between these two properties by prosa text should suffice.